.env.test

```diff
@@ -1,7 +1,7 @@
 # Specific environment variables when running tests
 # Make sure to sync this file with turbo.json so that CI can pass
 NODE_ENV=test
-DATABASE_URL=postgres://root:root@localhost:5432/test
+DATABASE_URL=postgres://root:root@localhost:5432/app
```

Comment:
was this change intentional?

Reply:
Yeap! Oddly enough, it was trying to connect to `test` and failing. This change made it run on local but I think this shouldn't be needed because we mock prisma w pglite. Need to investigate more